zkevm-circuits/src/evm_circuit.rs

```diff
@@ -1296,4 +1302,87 @@ mod test {
                 .assign(&mut layouter, &self.execution_steps)
         }
     }
+
+    // To run this benchmark, comment the `ignore` flag and run the following
+    // command:
+    // `RUSTFLAGS='-C target-cpu=native' cargo test --profile bench
+    // bench_evm_circuit_prover -- --nocapture`
+    #[ignore]
+    #[test]
+    fn bench_evm_circuit_prover() {
+        const DEGREE: u32 = 22;
+        const EXECUTION_STEPS_ROWS_MAX: usize = 1 << (DEGREE - 2);
+        const OPERATIONS_ROWS_MAX: usize = 1 << (DEGREE - 2);
```

Comment:
@han0110 is it correct to leave this as is? 
I needed to add that in the `state_circuit` since otherways we were getting a `DegreeBounds` err.

Reply:
I would suggest to just pass empty `execution_steps` and `operations` to avoid debugging since the witness assignment time is negligible compared to proof calculation time.

But for a more precise value, the max `EXECUTION_STEPS_ROWS_MAX` should be `(1 << (DEGREE - 1)) / STEP_HEIGHT`, and the `OPERATIONS_ROWS_MAX` should be `1 << (DEGREE-1)`

Reply:
Thanks a lot!!
@NoCtrlZ I think we then should proceed on that way.